src/math/tests/evaluate.rs

```diff
@@ -0,0 +1,107 @@
+use math::evaluate;
+use math::number::Radix;
+
+fn eval_dec(s: &str, precision: u8) -> Result<String, math::error::Error> {
```

Comment:
In case you don't know, this return type can be written as `math::Result<String>`

Reply:
Oh right, thanks!